flink-table/flink-table-api-java-bridge/src/test/java/org/apache/flink/table/factories/DataGenTableSourceFactoryTest.java

```diff
@@ -581,6 +588,27 @@ void testWrongTypes() {
                         anyCauseMatches("Could not parse value 'Wrong' for key 'fields.f0.start'"));
     }
 
+    @Test
+    void testWithParallelism() {
```

Comment:
In FLINK-33263, we've introduced a way to verify that the transformation, could you also add a test based on that?

Reply:
I've already included an IT case in `DataGeneratorConnectorITCase#testWithParallelism` where the physical execution plan is checked, would that be enough?